index.js

```diff
@@ -35,18 +35,18 @@ exports.clean = async function() {
 // Only works for dev environments
 exports.createDevAccount = async function(options) {
     const keyPair = await KeyPair.fromRandomSeed();
-
+    let accountId = options.accountId;
```

Comment:
nit: `const { accountId } = options;` (same tricks can be used to extract other options)

Reply:
Ah yeah, good call. Will add.